src/components/Login/LoginForm.tsx

```diff
@@ -46,21 +50,31 @@ const LoginForm = ({ addNewWallet }: LoginFormProps) => {
   const [error, setError] = useState(null);
   const [feedback, setFeedback] = useState('');
   const [showSpinner, setShowSpinner] = useState(false);
+  const loginFeedback = useSelector(selectLoginFeedback);
+  const envError = useSelector(selectEnvError);
 
   const performLogin = useCallback(() => {
+    dispatch.environment.SET_ERROR(null);
     if (!loginValue) {
       setError(ErrorMessages.ENTER_WIF);
       return;
     }
-    setShowSpinner(true);
-    dispatch.account.login({ key: loginValue, setError, setFeedback });
+    ipcRenderer.send(BITGO, login(loginValue));
   }, [loginValue]);
 
   useEffect(() => {
-    if (error) {
+    setError(envError);
+    if (error || envError) {
       setShowSpinner(false);
     }
-  }, [error]);
+  }, [error, envError]);
+
+  useEffect(() => {
```

Comment:
Is this necessary? `setFeedback` seems to just be copying thru data we have in the redux for no apparent reason. Just use `loginFeedback` directly instead of copying it into local state. And then in the render we can just show the spinner if any of the data is missing (or not show if anything errors out).